scania-digital-design-system/sdds
# fix(sidemenu): move theming rules to component

**Describe pull-request** 
Removing theming rules in theme light. They are not used in component.

**Solving issue** 
Fixes: [AB#2220](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2220) 

**How to test** 
1. Open sidemenu in Storybook of this branch 
2. Open sidemenu in Storybook of production
3.  3. Compare styling- there should not be any difference between those two
